pyproject.toml

```diff
@@ -55,7 +55,7 @@ influxdb = "^5.3.1"
 # required for tests to pass right now. There are often issues when pinning to a branch
 # and calling poetry update. I have had to completely remove my venv and reinstall in order
 # to get the latest updates.
-alfalfa-client = "^0.4.0"
+alfalfa-client = {git = "https://git@github.com/nrel/alfalfa-client.git", branch = "rest_api"}
```

Comment:
We'll be changing this after merging, right?

Reply:
Yeah, the alfalfa client change to rest API hasn't been merged or released on that side yet